app/models/organization.rb

```diff
@@ -203,6 +203,16 @@ def valid_items
     end
   end
 
+  def item_id_to_display_string_map
+    valid_items.each_with_object({}) do |item, hash|
+      hash[item[:id].to_i] = item[:name]
+    end
+  end
+
+  def valid_items_for_select
+    valid_items.map { |item| [item[:name], item[:id]] }.sort
+  end
```

Comment:
Let's make sure to add specs for these in another PR. I feel like these are crucial and is important that it works :)